index.html

```diff
@@ -1,179 +1,250 @@
+                <button type="button" class="theButton">See Project</button>
+              </section>
+            </section>
+            <section class="other">
+              <section class="informationCard">
+                <h3 class="subTitle">Multi-Post Stories Gaing+Glory</h3> 
+                <ul>
+                  <li class="myRecentWorks">Ruby on rails</li>
+                  <li class="myRecentWorks">css</li>
+                  <li class="myRecentWorks">JavaScript</li>
+                  <li class="myRecentWorks">html</li>
+                </ul>
+                <button type="button" class="theButton">See Project</button>
+              </section>
+            </section>
+            <section class="other">
+              <section class="informationCard">
+                <h3 class="subTitle">Multi-Post Stories Gaing+Glory</h3> 
+                <ul>
+                  <li class="myRecentWorks">Ruby on rails</li>
+                  <li class="myRecentWorks">css</li>
+                  <li class="myRecentWorks">JavaScript</li>
+                  <li class="myRecentWorks">html</li>
+                </ul>
+                <button type="button" class="theButton">See Project</button>
+              </section>
+            </section>
+          </div>  
+        </section>
+        <section class="about-me">
+          <img class="img1-aboutme" src="./imges/aboutme-img1.png">
+          <h2 class="color-blue">About me</h2>
+          <p class="color-blue">
+            I can help you build a product , feature or website Look through some 
+            of my work and experience! If you like what you see and have a project 
+            you need coded, don&apos;t hestiate to contact me.
+          </p>
+            <button type="button" class="btn-cv">Get My Resume</button>
+        </section>
+        <section class="skills-card">
+          <div class="img2-aboutme">
+            <section class="postOne">
+              <section class="my-placeholder">
+                <img src="./imges/Icon Name.svg" alt="skill"/>
+                <h3 class="subTitle">Languages</h3>
+                <ul>
+                  <li class="aboutMeListItem">Javascript</li>
+                  <li class="aboutMeListItem">Ruby</li>
+                  <li class="aboutMeListItem">Html</li>
+                  <li class="aboutMeListItem">CSS</li>
+                </ul>
+              </section>
+            </section>
+            <section class="postOne">
+              <section class="my-placeholder">
+                <img src="./imges/Group.svg" alt="skill"/>
+                <h3 class="subTitle">Frameworks</h3>
+                <ul>
+                  <li class="aboutMeListItem">Bootstrap</li>
+                  <li class="aboutMeListItem">Ruby on rails</li>
+                  <li class="aboutMeListItem">Rspec</li>
+                  <li class="aboutMeListItem">Capybara</li>
+                  <li class="aboutMeListItem">Selenium</li>
+                </ul>
```

Comment:
- [ ] [Optional] I am giving you an option because this is the last review for you. please implement it. Please decrease the spacing between tools and skills to stick to the design. See image below for reference
### Your design
![sssssssssssssss](https://user-images.githubusercontent.com/81727988/201500029-d09406ab-43fd-46d4-9183-3cd16829ebac.png)